R/query.R

```diff
@@ -80,6 +82,9 @@ do_query <- function(query, server.url = NULL, timeout = NULL, print.json = FALS
         else
             ret <- convert_query_results(query, res.data.content$query_result, ...)
 
+    } else if (httr::status_code(response) == 401) {
+        stop(paste("Not authorized. If you have already login using wick::login, your authorization token may be expired.",
```

Comment:
I'm not sure we should specifically reference wick::login here since this package is open source and could be used from other platforms